client/jetpack-cloud/sections/partner-portal/hooks.tsx

```diff
@@ -477,3 +477,84 @@ export function useAssignMultipleLicenses(
 	const [ assign, isLoading ] = useAssignLicenses( selectedLicenseKeys, selectedSite );
 	return [ assign, isLoading ];
 }
+
+/**
+ * Returns product description and features with given product slug.
+ *
+ * @param productSlug
+ * @returns
+ */
+export function useProductDescription( productSlug: string ): {
+	description: TranslateResult | null;
+	features: ReadonlyArray< any >;
+} {
+	const translate = useTranslate();
+
+	return useMemo( () => {
+		let description = '';
+		const features = [];
+
+		switch ( productSlug ) {
+			case 'jetpack-complete':
+				description = translate( 'Includes all Security 1TB and full Jetpack package.' );
+				features.push(
+					translate( 'All Security products' ),
+					translate( '1TB cloud storage' ),
+					translate( 'Full Jetpack package' )
+				);
+				break;
+			case 'jetpack-security-t1':
+				description = translate(
+					'Includes VaultPress Backup 10GB, Scan Daily and Akismet Anti-spam.'
+				);
+				features.push(
+					translate( 'VaultPress Backup 10GB' ),
+					translate( 'Scan Daily' ),
+					translate( 'Akismet Anti-spam*' )
+				);
+				break;
+			case 'jetpack-security-t2':
+				description = translate(
+					'Includes VaultPress Backup 1TB, Scan Daily and Akismet Anti-spam.'
+				);
+				features.push(
+					translate( 'VaultPress Backup 1TB' ),
+					translate( 'Scan Daily' ),
+					translate( 'Akismet Anti-spam*' )
+				);
+				break;
+			case 'jetpack-anti-spam':
+				description = translate( 'Automatically clear spam from your comments and forms.' );
+				break;
+			case 'jetpack-backup-t1':
+				description = translate( 'Real-time cloud backups with one-click restores.' );
+				break;
+			case 'jetpack-backup-t2':
+				description = translate( 'Real-time cloud backups with one-click restores.' );
+				break;
```

Comment:
If descriptions of these two products will always be the same, it will be better to group the similar cases to avoid some code duplication.

```suggestion
			case 'jetpack-backup-t1':
			case 'jetpack-backup-t2':
				description = translate( 'Real-time cloud backups with one-click restores.' );
				break;
```

Reply:
Good catch, fixed this one 👍